src/absil/illib.fs

```diff
@@ -340,7 +340,7 @@ module List =
           | _ -> finished <- true
         List.rev r, s
 
-    // note: not tail recursive 
+    // NOTE: not tail recursive 
```

Comment:
Ditto Note:  Last one I'm going to mention, if you see one in a comment it is redundant by definition :-)